go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/vmware-tanzu/vm-operator-api v0.1.4-0.20211202183846-992b48c128ae
 	github.com/vmware-tanzu/vm-operator/api v1.8.2
-	github.com/vmware/govmomi v0.33.0
```

Comment:
Can you make sure to request new govomi release or use the govmomi commit https://github.com/vmware/govmomi/commit/2ed0a9d608a690c3aeeb66bee470d7964bfdcc9f

Reply:
New govmomi release v0.34.2 has been created